hybrid/HybridSampleApps/AccountEditor/AndroidManifest.xml

```diff
@@ -23,12 +23,39 @@
         <!--  Main activity -->
         <activity android:label="@string/app_name"
             android:name="com.salesforce.androidsdk.phonegap.ui.SalesforceDroidGapActivity"
-            android:configChanges="orientation|keyboardHidden" >
+            android:configChanges="orientation|keyboardHidden">
+
             <intent-filter >
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+
+        <!-- IDP account picker activity -->
+        <!--
+            To enable this app as an IDP app that's capable of handling authentication
+            requests originating from other SP apps, uncomment the lines below and replace
+            'scheme' and 'host' with the values you wish to use to define your IDP app.
+            The URI constructed from this must be unique and will be set in the SP app
+            as well. This will be used to launch the IDP app from the SP app.
+
+            In the example given below, 'scheme' is "accounteditor" and 'host' is "authenticator".
+            The full URI that needs to be set on the SP app would be "accounteditor://authenticator".
+        -->
+        <!--
+        <activity android:name="com.salesforce.androidsdk.auth.idp.IDPAccountPickerActivity"
```

Comment:
This needs to be uncommented to make this app behave as an IDP. I have given each sample app its own custom scheme based on the app name. For example, this one uses `accounteditor://authenticator`.

Reply:
Should the custom scheme be based on the package name (e.g. com.salesforce.samples.accounteditor)?

Reply:
Yeah - makes sense. I'll make the change.